package.json

```diff
@@ -35,11 +42,11 @@
     "preview:local": "VITE_TEMPORAL_UI_BUILD_TARGET=local vite preview --port 3000",
     "preview:docker": "VITE_API=http://localhost:8080 VITE_TEMPORAL_UI_BUILD_TARGET=local vite preview --port 3000",
     "preview:cloud": "VITE_TEMPORAL_UI_BUILD_TARGET=cloud vite preview --port 3000",
-    "package": "svelte-package && rsync -r ./package/lib/ ./package/ && rm -rf ./package/lib",
+    "package": "svelte-package",
     "package:patch": "pnpm version patch && svelte-package",
     "package:minor": "pnpm version minor && svelte-package",
     "package:major": "pnpm version major && svelte-package",
-    "story:dev": "histoire dev",
+    "story:dev": "histoire dev --port 3030",
```

Comment:
why the port change?

Reply:
I was on an airplane and the console outputted so much stuff I couldn't find the port it was running on and I didn't have internet so I just set it I'll revert.